acceptance/auth/bundle_and_profile/output.txt

```diff
@@ -0,0 +1,32 @@
+
+=== Inside the bundle, no flags
+>>> errcode [CLI] current-user me
+"[USERNAME]"
```

Comment:
I suspect this works without the `databricks.yml` as well, because `$DATABRICKS_HOST` is set in the environment. You could either unset those env vars, or force it to be a different server as suggested in the other comment.